app/models/team.rb

```diff
@@ -198,11 +198,11 @@ def import_team_members(row_hash)
     end
   end
 
-  #  changed to hash by E1776
-  def self.import(row_hash, id, options, teamtype)
-    raise ArgumentError, 'Not enough fields on this line.' if row_hash.empty? || (row_hash[:teammembers].empty? && (options[:has_teamname] == 'true_first' || options[:has_teamname] == 'true_last')) || (row_hash[:teammembers].empty? && (options[:has_teamname] == 'true_first' || options[:has_teamname] == 'true_last'))
-
-    if options[:has_teamname] == 'true_first' || options[:has_teamname] == 'true_last'
+  # Helper for importing CourseTeam and AssignmentTeam objects. Should not be used to import
+  # pure Team objects.
```

Comment:
There are no Teams that are not assignment or course teams.  So this comment is unnecessary.

Reply:
Removed.